spec/index.bs

```diff
@@ -990,7 +990,7 @@ Given a {{TrustedType}} type (|expectedType|), a [=realm/global object=] (|globa
 1.  Let |convertedInput| be the result of executing [$Create a
     Trusted Type$] algorithm, with the following arguments:
     *  |defaultPolicy| as |policy|
-    *  |input| as |value|
+    *  |input| as |value|, stringifying if |input| is a {{TrustedType}}
```

Comment:
You could also just say `stringified |input| as |value|`, stringifying a string is a noop.